packages/components/src/components/chip/chip-item/chip-item.tsx

```diff
@@ -0,0 +1,158 @@
+import { h,
+         Component,
+         Element,
+         Event,
+         EventEmitter, 
+         Listen,
+         Prop, 
+         Watch } from '@stencil/core';
+import { ChipItemEvent, ChipState } from '../interfaces';
+
+@Component({
+    tag: 'ifx-chip-item',
+    styleUrl: 'chip-item.scss',
+    shadow: true
+})
+
+export class ChipItem {
+    /**
+     * Reference to this component.
```

Comment:
would also remove all these comments above the variable names, as the variable names should be self explanatory